src/openms/include/OpenMS/ANALYSIS/MRM/ReactionMonitoringTransition.h

```diff
@@ -154,6 +154,12 @@ namespace OpenMS
 
     void setLibraryIntensity(double intensity);
 
+    bool isDetectingTransition() const;
```

Comment:
I think you should write some documentation here what this really means, e.g. what is the idea behind detecting/identifying and how should these transitions be used? I think this would be helpful to have that written down somewhere